dcpy/lifecycle/ingest/scripts/doe_pepmeetingurls.py

```diff
@@ -109,10 +107,3 @@ def ingest() -> pd.DataFrame:
         proposals1.append(g1)
     df = pd.DataFrame(proposals1)
     return df
-
-
-def runner(dir=TMP_DIR) -> Path:
-    df = ingest()
-    path = dir / "doe_pepmeetingurls.csv"
-    df.to_csv(path, index=False)
-    return path
```

Comment:
Decided it made more sense for the "script" source to just return data, not convert it to a file. This could maybe change over time, this isn't really working or tested yet. Want the rest of things to settle out (and figure out how many real "script" sources we'll have) before refining. Per a comment from @alexrichey a while back, these maybe will just become their own connectors. We'll see